.github/workflows/continuous-deployment.yml

```
@@ -117,6 +117,10 @@ jobs:
         run: |
           pnpm run --if-present build
 
+      - name: Check the release
+        run: |
+          pnpm run --if-present lint-release
```

```suggestion
        run: pnpm run lint-release
```

De conventie die we overal hebben is enkele commando's op 1 regel, meerdere commando's pas over meerdere regels met een `run: |`

Die `--if-present` is ook niet nodig… ik denk dat we juist willen weten dat er iets niet meer nodig is als we het weggehaald hebben.